lib/knapsack_pro/adapters/rspec_adapter.rb

```diff
@@ -20,22 +20,17 @@ def self.split_by_test_cases_enabled?
         true
       end
 
-      def self.test_file_cases_for(slow_test_files)
-        KnapsackPro.logger.info("Generating RSpec test examples JSON report for slow test files to prepare it to be split by test examples (by individual test cases). Thanks to that, a single slow test file can be split across parallel CI nodes. Analyzing #{slow_test_files.size} slow test files.")
```

Comment:
suggestion

We could keep a count of how many tests were slow in the logs.

Having `Analyzing #{slow_test_files.size} slow test files.` in logs was useful whenever we did support and needed to check if any test file was split by examples.